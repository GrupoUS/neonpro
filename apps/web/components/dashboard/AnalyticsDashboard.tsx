/**
 * Analytics Dashboard Component
 * Healthcare platform analytics visualization
 */

export interface AnalyticsDashboardProps {
  data?: unknown
  loading?: boolean
  error?: string
}

export default function AnalyticsDashboard({
  data,
  loading = false,
  error,
}: AnalyticsDashboardProps,) {
  if (loading) {
    return <div>Loading analytics...</div>
  }

  if (error) {
    return <div>Error loading analytics: {error}</div>
  }

  return (
    <div className="analytics-dashboard">
      <h2>Healthcare Analytics Dashboard</h2>
      <div className="metrics-grid">
        {/* Analytics content would go here */}
        <div>Analytics data placeholder</div>
      </div>
    </div>
  )
}
