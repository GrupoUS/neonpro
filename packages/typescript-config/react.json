{
  "extends": "./base.json",
  "compilerOptions": {
    "jsx": "react-jsx",
    "lib": ["ES2022", "DOM", "DOM.Iterable"]
  },
  "include": ["src/**/*.ts", "src/**/*.tsx", "src/**/*.d.ts"]
}