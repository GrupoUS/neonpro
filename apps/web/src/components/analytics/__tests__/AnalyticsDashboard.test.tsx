import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import { AnalyticsDashboard } from '../AnalyticsDashboard';

describe(('AnalyticsDashboard', () => {
  it(('renders correctly - snapshot test', () => {
    const { container } = render(<AnalyticsDashboard />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it(('renders header section with title and description', () => {
    const { getByText } = render(<AnalyticsDashboard />);

    expect(getByText('Analytics Dashboard')).toBeInTheDocument();
    expect(
      getByText('Análise inteligente de dados clínicos e operacionais'),
    ).toBeInTheDocument();
  });

  it(('renders KPI cards with skeleton placeholders', () => {
    const { getByText } = render(<AnalyticsDashboard />);

    expect(getByText('Taxa de Presença')).toBeInTheDocument();
    expect(getByText('Satisfação do Cliente')).toBeInTheDocument();
    expect(getByText('Receita por Paciente')).toBeInTheDocument();
    expect(getByText('Eficiência Operacional')).toBeInTheDocument();
  });

  it(('renders trends chart placeholder', () => {
    const { getByText } = render(<AnalyticsDashboard />);

    expect(getByText('Tendências de Agendamento')).toBeInTheDocument();
    expect(
      getByText('Gráfico de tendências será exibido aqui'),
    ).toBeInTheDocument();
  });

  it(('renders AI insights section', () => {
    const { getByText } = render(<AnalyticsDashboard />);

    expect(getByText('Insights de IA')).toBeInTheDocument();
    expect(getByText('Padrão de Cancelamentos Detectado')).toBeInTheDocument();
    expect(getByText('Oportunidade de Upselling')).toBeInTheDocument();
    expect(getByText('Capacidade Ociosa Identificada')).toBeInTheDocument();
  });

  it(('renders performance metrics with progress bars', () => {
    const { getByText } = render(<AnalyticsDashboard />);

    expect(getByText('Métricas de Performance')).toBeInTheDocument();
    expect(getByText('Taxa de Ocupação')).toBeInTheDocument();
    expect(getByText('Tempo Médio de Espera')).toBeInTheDocument();
    expect(getByText('NPS Score')).toBeInTheDocument();
    expect(getByText('Taxa de Retorno')).toBeInTheDocument();
  });

  it(('renders quick actions buttons', () => {
    const { getByText } = render(<AnalyticsDashboard />);

    expect(getByText('Ações Rápidas')).toBeInTheDocument();
    expect(getByText('Relatório de Agendamentos')).toBeInTheDocument();
    expect(getByText('Análise de Pacientes')).toBeInTheDocument();
    expect(getByText('Relatório Financeiro')).toBeInTheDocument();
    expect(getByText('Previsões de IA')).toBeInTheDocument();
    expect(getByText('Análise de Eficiência')).toBeInTheDocument();
  });

  it(('renders data status section', () => {
    const { getByText } = render(<AnalyticsDashboard />);

    expect(getByText('Status dos Dados')).toBeInTheDocument();
    expect(getByText('Última sincronização')).toBeInTheDocument();
    expect(getByText('Registros processados')).toBeInTheDocument();
    expect(getByText('Qualidade dos dados')).toBeInTheDocument();
  });

  it(('applies custom className when provided', () => {
    const { container } = render(
      <AnalyticsDashboard className='custom-class' />,
    );

    expect(container.firstChild).toHaveClass('custom-class');
  });
});
