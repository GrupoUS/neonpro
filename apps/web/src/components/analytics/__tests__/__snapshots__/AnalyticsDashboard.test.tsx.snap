// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AnalyticsDashboard > renders correctly - snapshot test 1`] = `
<div
  class="space-y-6 undefined"
>
  <div
    class="flex items-center justify-between"
  >
    <div>
      <h2
        class="text-3xl font-bold tracking-tight"
      >
        Analytics Dashboard
      </h2>
      <p
        class="text-muted-foreground"
      >
        Análise inteligente de dados clínicos e operacionais
      </p>
    </div>
    <button
      class="inline-flex items-center justify-center whitespace-nowrap text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 border border-input bg-background hover:bg-accent hover:text-accent-foreground h-9 rounded-md px-3"
    >
      <svg
        aria-hidden="true"
        class="lucide lucide-refresh-cw h-4 w-4 mr-2"
        fill="none"
        height="24"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M3 12a9 9 0 0 1 9-9 9.75 9.75 0 0 1 6.74 2.74L21 8"
        />
        <path
          d="M21 3v5h-5"
        />
        <path
          d="M21 12a9 9 0 0 1-9 9 9.75 9.75 0 0 1-6.74-2.74L3 16"
        />
        <path
          d="M8 16H3v5"
        />
      </svg>
      Atualizar Dados
    </button>
  </div>
  <div
    class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-4"
  >
    <div
      class="rounded-lg border bg-card text-card-foreground shadow-sm"
    >
      <div
        class="p-6 flex flex-row items-center justify-between space-y-0 pb-2"
      >
        <h3
          class="tracking-tight text-sm font-medium"
        >
          Taxa de Presença
        </h3>
        <svg
          aria-hidden="true"
          class="lucide lucide-trending-up h-4 w-4 text-muted-foreground"
          fill="none"
          height="24"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16 7h6v6"
          />
          <path
            d="m22 7-8.5 8.5-5-5L2 17"
          />
        </svg>
      </div>
      <div
        class="p-6 pt-0"
      >
        <div
          class="animate-pulse rounded-md bg-muted h-8 w-16 mb-2"
        />
        <p
          class="text-xs text-muted-foreground"
        >
          +2.5% em relação ao mês anterior
        </p>
      </div>
    </div>
    <div
      class="rounded-lg border bg-card text-card-foreground shadow-sm"
    >
      <div
        class="p-6 flex flex-row items-center justify-between space-y-0 pb-2"
      >
        <h3
          class="tracking-tight text-sm font-medium"
        >
          Satisfação do Cliente
        </h3>
        <svg
          aria-hidden="true"
          class="lucide lucide-users h-4 w-4 text-muted-foreground"
          fill="none"
          height="24"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16 21v-2a4 4 0 0 0-4-4H6a4 4 0 0 0-4 4v2"
          />
          <path
            d="M16 3.128a4 4 0 0 1 0 7.744"
          />
          <path
            d="M22 21v-2a4 4 0 0 0-3-3.87"
          />
          <circle
            cx="9"
            cy="7"
            r="4"
          />
        </svg>
      </div>
      <div
        class="p-6 pt-0"
      >
        <div
          class="animate-pulse rounded-md bg-muted h-8 w-16 mb-2"
        />
        <p
          class="text-xs text-muted-foreground"
        >
          Baseado em 127 avaliações
        </p>
      </div>
    </div>
    <div
      class="rounded-lg border bg-card text-card-foreground shadow-sm"
    >
      <div
        class="p-6 flex flex-row items-center justify-between space-y-0 pb-2"
      >
        <h3
          class="tracking-tight text-sm font-medium"
        >
          Receita por Paciente
        </h3>
        <svg
          aria-hidden="true"
          class="lucide lucide-dollar-sign h-4 w-4 text-muted-foreground"
          fill="none"
          height="24"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <line
            x1="12"
            x2="12"
            y1="2"
            y2="22"
          />
          <path
            d="M17 5H9.5a3.5 3.5 0 0 0 0 7h5a3.5 3.5 0 0 1 0 7H6"
          />
        </svg>
      </div>
      <div
        class="p-6 pt-0"
      >
        <div
          class="animate-pulse rounded-md bg-muted h-8 w-20 mb-2"
        />
        <p
          class="text-xs text-muted-foreground"
        >
          +12.3% vs média anual
        </p>
      </div>
    </div>
    <div
      class="rounded-lg border bg-card text-card-foreground shadow-sm"
    >
      <div
        class="p-6 flex flex-row items-center justify-between space-y-0 pb-2"
      >
        <h3
          class="tracking-tight text-sm font-medium"
        >
          Eficiência Operacional
        </h3>
        <svg
          aria-hidden="true"
          class="lucide lucide-activity h-4 w-4 text-muted-foreground"
          fill="none"
          height="24"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M22 12h-2.48a2 2 0 0 0-1.93 1.46l-2.35 8.36a.25.25 0 0 1-.48 0L9.24 2.18a.25.25 0 0 0-.48 0l-2.35 8.36A2 2 0 0 1 4.49 12H2"
          />
        </svg>
      </div>
      <div
        class="p-6 pt-0"
      >
        <div
          class="animate-pulse rounded-md bg-muted h-8 w-16 mb-2"
        />
        <p
          class="text-xs text-muted-foreground"
        >
          Tempo médio por consulta
        </p>
      </div>
    </div>
  </div>
  <div
    class="grid grid-cols-1 lg:grid-cols-2 gap-6"
  >
    <div
      class="space-y-6"
    >
      <div
        class="rounded-lg border bg-card text-card-foreground shadow-sm"
      >
        <div
          class="flex flex-col space-y-1.5 p-6"
        >
          <h3
            class="text-2xl font-semibold leading-none tracking-tight flex items-center gap-2"
          >
            <svg
              aria-hidden="true"
              class="lucide lucide-chart-column h-5 w-5"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M3 3v16a2 2 0 0 0 2 2h16"
              />
              <path
                d="M18 17V9"
              />
              <path
                d="M13 17V5"
              />
              <path
                d="M8 17v-3"
              />
            </svg>
            Tendências de Agendamento
          </h3>
          <p
            class="text-sm text-muted-foreground"
          >
            Análise temporal dos agendamentos e comparação com períodos anteriores
          </p>
        </div>
        <div
          class="p-6 pt-0"
        >
          <div
            class="h-64 bg-muted rounded-lg flex items-center justify-center"
          >
            <div
              class="text-center space-y-2"
            >
              <svg
                aria-hidden="true"
                class="lucide lucide-chart-column h-12 w-12 text-muted-foreground mx-auto"
                fill="none"
                height="24"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M3 3v16a2 2 0 0 0 2 2h16"
                />
                <path
                  d="M18 17V9"
                />
                <path
                  d="M13 17V5"
                />
                <path
                  d="M8 17v-3"
                />
              </svg>
              <p
                class="text-sm text-muted-foreground"
              >
                Gráfico de tendências será exibido aqui
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="rounded-lg border bg-card text-card-foreground shadow-sm"
      >
        <div
          class="flex flex-col space-y-1.5 p-6"
        >
          <h3
            class="text-2xl font-semibold leading-none tracking-tight flex items-center gap-2"
          >
            <svg
              aria-hidden="true"
              class="lucide lucide-brain h-5 w-5"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M12 18V5"
              />
              <path
                d="M15 13a4.17 4.17 0 0 1-3-4 4.17 4.17 0 0 1-3 4"
              />
              <path
                d="M17.598 6.5A3 3 0 1 0 12 5a3 3 0 1 0-5.598 1.5"
              />
              <path
                d="M17.997 5.125a4 4 0 0 1 2.526 5.77"
              />
              <path
                d="M18 18a4 4 0 0 0 2-7.464"
              />
              <path
                d="M19.967 17.483A4 4 0 1 1 12 18a4 4 0 1 1-7.967-.517"
              />
              <path
                d="M6 18a4 4 0 0 1-2-7.464"
              />
              <path
                d="M6.003 5.125a4 4 0 0 0-2.526 5.77"
              />
            </svg>
            Insights de IA
          </h3>
          <p
            class="text-sm text-muted-foreground"
          >
            Recomendações inteligentes baseadas em padrões de dados
          </p>
        </div>
        <div
          class="p-6 pt-0"
        >
          <div
            class="space-y-4"
          >
            <div
              class="flex items-start gap-3"
            >
              <div
                class="w-2 h-2 bg-blue-500 rounded-full mt-2"
              />
              <div
                class="flex-1"
              >
                <p
                  class="text-sm font-medium"
                >
                  Padrão de Cancelamentos Detectado
                </p>
                <p
                  class="text-xs text-muted-foreground"
                >
                  Terças-feiras têm 23% mais cancelamentos. Considere implementar confirmações automáticas.
                </p>
                <div
                  class="inline-flex items-center rounded-full border px-2.5 py-0.5 text-xs font-semibold transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2 border-transparent bg-secondary text-secondary-foreground hover:bg-secondary/80 mt-1"
                >
                  Recomendação
                </div>
              </div>
            </div>
            <div
              class="flex items-start gap-3"
            >
              <div
                class="w-2 h-2 bg-green-500 rounded-full mt-2"
              />
              <div
                class="flex-1"
              >
                <p
                  class="text-sm font-medium"
                >
                  Oportunidade de Upselling
                </p>
                <p
                  class="text-xs text-muted-foreground"
                >
                  45% dos pacientes de limpeza de pele retornam em 30 dias. Ideal para pacotes.
                </p>
                <div
                  class="inline-flex items-center rounded-full border px-2.5 py-0.5 text-xs font-semibold transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2 border-transparent bg-secondary text-secondary-foreground hover:bg-secondary/80 mt-1"
                >
                  Oportunidade
                </div>
              </div>
            </div>
            <div
              class="flex items-start gap-3"
            >
              <div
                class="w-2 h-2 bg-orange-500 rounded-full mt-2"
              />
              <div
                class="flex-1"
              >
                <p
                  class="text-sm font-medium"
                >
                  Capacidade Ociosa Identificada
                </p>
                <p
                  class="text-xs text-muted-foreground"
                >
                  Quintas 14h-16h têm baixa ocupação. Considere promoções direcionadas.
                </p>
                <div
                  class="inline-flex items-center rounded-full border px-2.5 py-0.5 text-xs font-semibold transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2 border-transparent bg-secondary text-secondary-foreground hover:bg-secondary/80 mt-1"
                >
                  Otimização
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="space-y-6"
    >
      <div
        class="rounded-lg border bg-card text-card-foreground shadow-sm"
      >
        <div
          class="flex flex-col space-y-1.5 p-6"
        >
          <h3
            class="text-2xl font-semibold leading-none tracking-tight flex items-center gap-2"
          >
            <svg
              aria-hidden="true"
              class="lucide lucide-target h-5 w-5"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                cx="12"
                cy="12"
                r="10"
              />
              <circle
                cx="12"
                cy="12"
                r="6"
              />
              <circle
                cx="12"
                cy="12"
                r="2"
              />
            </svg>
            Métricas de Performance
          </h3>
          <p
            class="text-sm text-muted-foreground"
          >
            Indicadores chave de performance da clínica
          </p>
        </div>
        <div
          class="p-6 pt-0"
        >
          <div
            class="space-y-4"
          >
            <div
              class="flex items-center justify-between"
            >
              <span
                class="text-sm"
              >
                Taxa de Ocupação
              </span>
              <div
                class="flex items-center gap-2"
              >
                <div
                  class="w-24 bg-muted rounded-full h-2"
                >
                  <div
                    class="bg-blue-500 h-2 rounded-full"
                    style="width: 78%;"
                  />
                </div>
                <span
                  class="text-sm font-medium"
                >
                  78%
                </span>
              </div>
            </div>
            <div
              class="flex items-center justify-between"
            >
              <span
                class="text-sm"
              >
                Tempo Médio de Espera
              </span>
              <div
                class="flex items-center gap-2"
              >
                <div
                  class="w-24 bg-muted rounded-full h-2"
                >
                  <div
                    class="bg-green-500 h-2 rounded-full"
                    style="width: 65%;"
                  />
                </div>
                <span
                  class="text-sm font-medium"
                >
                  12min
                </span>
              </div>
            </div>
            <div
              class="flex items-center justify-between"
            >
              <span
                class="text-sm"
              >
                NPS Score
              </span>
              <div
                class="flex items-center gap-2"
              >
                <div
                  class="w-24 bg-muted rounded-full h-2"
                >
                  <div
                    class="bg-emerald-500 h-2 rounded-full"
                    style="width: 85%;"
                  />
                </div>
                <span
                  class="text-sm font-medium"
                >
                  8.5
                </span>
              </div>
            </div>
            <div
              class="flex items-center justify-between"
            >
              <span
                class="text-sm"
              >
                Taxa de Retorno
              </span>
              <div
                class="flex items-center gap-2"
              >
                <div
                  class="w-24 bg-muted rounded-full h-2"
                >
                  <div
                    class="bg-purple-500 h-2 rounded-full"
                    style="width: 72%;"
                  />
                </div>
                <span
                  class="text-sm font-medium"
                >
                  72%
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="rounded-lg border bg-card text-card-foreground shadow-sm"
      >
        <div
          class="flex flex-col space-y-1.5 p-6"
        >
          <h3
            class="text-2xl font-semibold leading-none tracking-tight flex items-center gap-2"
          >
            <svg
              aria-hidden="true"
              class="lucide lucide-zap h-5 w-5"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M4 14a1 1 0 0 1-.78-1.63l9.9-10.2a.5.5 0 0 1 .86.46l-1.92 6.02A1 1 0 0 0 13 10h7a1 1 0 0 1 .78 1.63l-9.9 10.2a.5.5 0 0 1-.86-.46l1.92-6.02A1 1 0 0 0 11 14z"
              />
            </svg>
            Ações Rápidas
          </h3>
          <p
            class="text-sm text-muted-foreground"
          >
            Análises e relatórios com um clique
          </p>
        </div>
        <div
          class="p-6 pt-0"
        >
          <div
            class="space-y-3"
          >
            <button
              class="inline-flex items-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 border border-input bg-background hover:bg-accent hover:text-accent-foreground h-10 px-4 py-2 w-full justify-start"
            >
              <svg
                aria-hidden="true"
                class="lucide lucide-calendar h-4 w-4 mr-2"
                fill="none"
                height="24"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M8 2v4"
                />
                <path
                  d="M16 2v4"
                />
                <rect
                  height="18"
                  rx="2"
                  width="18"
                  x="3"
                  y="4"
                />
                <path
                  d="M3 10h18"
                />
              </svg>
              Relatório de Agendamentos
            </button>
            <button
              class="inline-flex items-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 border border-input bg-background hover:bg-accent hover:text-accent-foreground h-10 px-4 py-2 w-full justify-start"
            >
              <svg
                aria-hidden="true"
                class="lucide lucide-users h-4 w-4 mr-2"
                fill="none"
                height="24"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M16 21v-2a4 4 0 0 0-4-4H6a4 4 0 0 0-4 4v2"
                />
                <path
                  d="M16 3.128a4 4 0 0 1 0 7.744"
                />
                <path
                  d="M22 21v-2a4 4 0 0 0-3-3.87"
                />
                <circle
                  cx="9"
                  cy="7"
                  r="4"
                />
              </svg>
              Análise de Pacientes
            </button>
            <button
              class="inline-flex items-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 border border-input bg-background hover:bg-accent hover:text-accent-foreground h-10 px-4 py-2 w-full justify-start"
            >
              <svg
                aria-hidden="true"
                class="lucide lucide-dollar-sign h-4 w-4 mr-2"
                fill="none"
                height="24"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <line
                  x1="12"
                  x2="12"
                  y1="2"
                  y2="22"
                />
                <path
                  d="M17 5H9.5a3.5 3.5 0 0 0 0 7h5a3.5 3.5 0 0 1 0 7H6"
                />
              </svg>
              Relatório Financeiro
            </button>
            <button
              class="inline-flex items-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 border border-input bg-background hover:bg-accent hover:text-accent-foreground h-10 px-4 py-2 w-full justify-start"
            >
              <svg
                aria-hidden="true"
                class="lucide lucide-trending-up h-4 w-4 mr-2"
                fill="none"
                height="24"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M16 7h6v6"
                />
                <path
                  d="m22 7-8.5 8.5-5-5L2 17"
                />
              </svg>
              Previsões de IA
            </button>
            <button
              class="inline-flex items-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 border border-input bg-background hover:bg-accent hover:text-accent-foreground h-10 px-4 py-2 w-full justify-start"
            >
              <svg
                aria-hidden="true"
                class="lucide lucide-activity h-4 w-4 mr-2"
                fill="none"
                height="24"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M22 12h-2.48a2 2 0 0 0-1.93 1.46l-2.35 8.36a.25.25 0 0 1-.48 0L9.24 2.18a.25.25 0 0 0-.48 0l-2.35 8.36A2 2 0 0 1 4.49 12H2"
                />
              </svg>
              Análise de Eficiência
            </button>
          </div>
        </div>
      </div>
      <div
        class="rounded-lg border bg-card text-card-foreground shadow-sm"
      >
        <div
          class="flex flex-col space-y-1.5 p-6"
        >
          <h3
            class="font-semibold tracking-tight text-sm"
          >
            Status dos Dados
          </h3>
        </div>
        <div
          class="p-6 pt-0"
        >
          <div
            class="space-y-2"
          >
            <div
              class="flex justify-between text-xs"
            >
              <span>
                Última sincronização
              </span>
              <span
                class="text-muted-foreground"
              >
                2 min atrás
              </span>
            </div>
            <div
              class="flex justify-between text-xs"
            >
              <span>
                Registros processados
              </span>
              <span
                class="text-muted-foreground"
              >
                1,247
              </span>
            </div>
            <div
              class="flex justify-between text-xs"
            >
              <span>
                Qualidade dos dados
              </span>
              <div
                class="inline-flex items-center rounded-full border px-2.5 py-0.5 font-semibold transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2 border-transparent bg-primary text-primary-foreground hover:bg-primary/80 text-xs"
              >
                95%
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
